CRM/Utils/Money.php

```diff
@@ -86,7 +86,7 @@ public static function format($amount, $currency = NULL, $format = NULL, $onlyNu
       CRM_Core_Error::deprecatedWarning('Passing empty currency to CRM_Utils_Money::format is deprecated if you need it for display without currency call CRM_Utils_Money::formatLocaleNumericRounded');
     }
 
-    $amount = self::formatNumericByFormat($amount);
+    $amount = self::formatUSLocaleNumericRounded($amount, 2);
```

Comment:
This is the actual change - the rest just renames a protected function to what it currently does